packages/workflow/src/WorkflowDataProxy.ts

```diff
@@ -105,7 +114,7 @@ export class WorkflowDataProxy {
 			ownKeys(target) {
 				return Reflect.ownKeys(target);
 			},
-			get(target, name, receiver) {
+			get(target, name) {
```

Comment:
We should double-check that the removed arg was unneeded.

Reply:
It was, thats why the linter pointed it out